### PR TITLE
Remove deprecated -teal image references

### LINKED ISSUE #326: Drop remaining "Elemental Teal" appearances
There's still "Elemental Teal" mentioned in the `1.4` docs, like in http://elemental.docs.rancher.com/quickstart-cli

We should find all these and replace them with "SLE Micro 5.5" (for 1.4) resp. "SL Micro 6.0" (for 1.6)

### PR DESCRIPTION
**Pull request by anmazzotti**
Closes #326

### CODE REVIEW

File: versioned_docs/version-1.4/architecture.md [hunk 1]
```diff
@@ -123,6 +123,6 @@ or node reboot.
 
 ----
 
-## Elemental Teal
+## Elemental
 
-Elemental Teal is Elemental OS built on top of SUSE Linux Enterprise (SLE) Micro for Rancher using the Elemental stack.
+Elemental OS is built on top of SUSE Linux Enterprise (SLE) Micro for Rancher using the Elemental stack.
```
Comment (kkaempf) [inline versioned_docs/version-1.4/architecture.md:128]: There's no "Elemental OS" - there's only SL(E) Micro as an OS.

File: versioned_docs/version-1.4/customizing.md [hunk 2]
```diff
@@ -275,7 +275,7 @@ show case how this could be added:
 
 ```docker showLineNumbers
 # The version of Elemental to modify
-FROM registry.suse.com/rancher/elemental-teal/5.4:latest
+FROM registry.suse.com/suse/sle-micro/5.5:2.0.2
```
Comment (kkaempf) [inline versioned_docs/version-1.4/customizing.md:278]: why not `5.5:latest` ?

File: versioned_docs/version-1.4/partials/_quickstart-prereqs.md [hunk 3]
```diff
@@ -6,7 +6,7 @@
   * Hint 1: Libvirt allows setting virtual TPMs for virtual machines [example here](tpm#add-tpm-module-to-virtual-machine)
   * Hint 2: You can enable TPM emulation on bare metal machines missing the TPM 2.0 module [example here](tpm#add-tpm-emulation-to-bare-metal-machine)
   * Hint 3: Make sure you're using UEFI (not BIOS) on x86-64, or the ISO won't boot
-  * Hint 4: A minimum volume size of 25 GB is recommended. See the [Elemental Teal partition table](installation#deployed-elemental-teal-partition-table) for more details  
+  * Hint 4: A minimum volume size of 25 GB is recommended. See the [Elemental Teal partition table](installation#deployed-elemental-partition-table) for more details
```
Comment (kkaempf) [inline versioned_docs/version-1.4/partials/_quickstart-prereqs.md:9]: There's still "Elemental Teal" here 😉

File: versioned_docs/version-1.4/rancher-vmware.md [hunk 4]
```diff
@@ -172,9 +172,9 @@ This command will download the vanilla iso and inject it with the parameters of
 If you would like to download the vanilla ISO and reuse it later to create additional ISO's, then you can download the iso separately using the below command and then pass the local file path as an argument to the script
 
 ```shell showLineNumbers
-wget https://download.opensuse.org/repositories/isv:/Rancher:/Elemental:/Stable:/Teal53/media/iso/elemental-teal.x86_64.iso 
+wget https://download.opensuse.org/repositories/isv:/Rancher:/Elemental:/Staging/containers/iso/sl-micro-6.0-baremetal.x86_64.iso
 
-./elemental-iso-add-registration initial-registration.yaml /home/elemental-iso/elemental-teal.x86_64.iso
+./elemental-iso-add-registration initial-registration.yaml /home/elemental-iso/sl-micro-6.0-baremetal.x86_64.iso
```
Comment (kkaempf) [inline versioned_docs/version-1.4/rancher-vmware.md:177]: No, please no mentioning of micro 6.0 before its release

**Review by kkaempf — CHANGES_REQUESTED** [review]

File: versioned_docs/version-1.4/customizing.md (hunk 2 above)
Comment (anmazzotti) [reply versioned_docs/version-1.4/customizing.md]: Too much copy paste, thank you.

File: versioned_docs/version-1.4/installation.md [hunk 5]
```diff
@@ -106,11 +104,11 @@ Once Elemental Teal is installed the OS partition table, according to default va
 
 Note this is the basic structure of any OS built by the <Vars name="elemental_toolkit_name" link="elemental_toolkit_url" />
 
-## Elemental Teal Immutable Root
+## Elemental Immutable Root
 
-One of the characteristics of Elemental OSes is the setup of an immutable root filesystem where some ephemeral or
-persistent locations are applied on top of it. Elemental Teal default folders structure is listed in the
-matrix below.
+ne of the characteristics of Elemental OSes is the setup of an immutable root
```
Comment (fgiudici) [inline versioned_docs/version-1.4/installation.md:109]: typo
```suggestion
One of the characteristics of Elemental OSes is the setup of an immutable root
```

**Review by fgiudici — APPROVED** [review]
looks good here!

**Pull request merged**